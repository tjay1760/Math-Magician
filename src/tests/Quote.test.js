import { render } from '@testing-library/react';
import Quote from '../components/DisplayQuote';

describe('Should render as expected', () => {
  test('test render of qoute page', () => {
    const { container } = render(<Quote />);
    expect(container).toMatchSnapshot();
  });
});
