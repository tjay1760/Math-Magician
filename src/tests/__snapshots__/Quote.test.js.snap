// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render as expected test render of qoute page 1`] = `
<div>
  <div>
    <p>
      Fetching Quote......
    </p>
  </div>
</div>
`;
